logging/fb/fluent-bit_config.configmap_azmonitor.yaml

```diff
@@ -900,20 +900,20 @@ data:
        Remove_regex   __temp_.+
     [FILTER]
        #
-       # WHITELIST Vars to output
+       # ALLOWLIST Vars to output
        #
```

Comment:
I wonder if we should add a comment like: 
`# Whitelist_key removed as alias for Allowlist_key in FB 1.9.x`
Or do we think the commit message is sufficient documentation?

Reply:
Ok, I'm with skipping the comment.